### PR TITLE
Add get_nwfilter_list as common util

### PR DESCRIPTION
**Pull request by chunfuwen**
Sometime user need nwfilter list from system

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
>>> cmd_result = '''
... setlocale: No such file or directory
...  UUID                                   Name
... -----------------------------------------------------------------
...  87c615b3-0f60-4c62-a092-abbce72ca40e   allow-arp
...  8f689e55-b7b3-4d12-a412-e6b09c36eb42   allow-dhcp
...  2c9b69fa-8ffe-4680-be65-0124764a8823   allow-dhcp-server
...  f047154c-a246-47ee-943a-c1ddd8af938a   allow-dhcpv6
...  06f19619-aaf6-4df0-8376-e15191183811   allow-dhcpv6-server
...  75e5a760-d61a-4fde-8615-f8c7787d8942   allow-incoming-ipv4
...  e7bbfe53-be00-4120-af51-2670d6b27d69   allow-incoming-ipv6
...  fc203183-4670-425a-a044-4d5438b0e2ac   allow-ipv4
...  674c627f-10f9-44d4-8357-c0a2b81de3d7   allow-ipv6
...  10717f8c-d719-4644-97bc-f3b8825ea5d2   clean-traffic
...  8cbb8b1b-645e-44c0-b171-aecabd8d96c3   clean-traffic-gateway
...  033bdd44-1f85-4abc-9cfd-10049c0ddb8c   no-arp-ip-spoofing
...  bcfcbb1e-693b-4a05-837d-61adebb72805   no-arp-mac-spoofing
...  3a10202a-7685-4a49-9aee-ab0b704fe63e   no-arp-spoofing
...  e42d0d2f-e768-49b1-8ce2-bd5fab8b9bdf   no-ip-multicast
...  192c79dd-0822-4792-9fb8-c2f132b3bc75   no-ip-spoofing
...  f6167dc8-d2a2-491a-b139-544cb454ceb6   no-ipv6-multicast
...  d2d8b32a-91b4-47f0-a15c-f2956ceb2560   no-ipv6-spoofing
...  b06561e1-6132-4748-b8df-59b17ead42a6   no-mac-broadcast
...  e8dedd25-53dc-41f9-8e3f-1deb3b96106a   no-mac-spoofing
...  8332c828-3f9e-4deb-b358-25e52c623271   no-other-l2-traffic
...  5b08c404-dcfe-45b4-b1a8-f92c84f5352a   no-other-rarp-traffic
...  3b1eab43-8976-4b48-99f3-05a645b8e8af   qemu-announce-self
...  b5aa9207-d9d7-4424-a088-7e2d51737743   qemu-announce-self-rarp
... '''

>>> uuid = re.findall(r"(\S+)\ +(\S+)", cmd_result.strip())
>>> uuid
[('setlocale:', 'No'), ('such', 'file'), ('or', 'directory'), ('UUID', 'Name'), ('87c615b3-0f60-4c62-a092-abbce72ca40e', 'allow-arp'), ('8f689e55-b7b3-4d12-a412-e6b09c36eb42', 'allow-dhcp'), ('2c9b69fa-8ffe-4680-be65-0124764a8823', 'allow-dhcp-server'), ('f047154c-a246-47ee-943a-c1ddd8af938a', 'allow-dhcpv6'), ('06f19619-aaf6-4df0-8376-e15191183811', 'allow-dhcpv6-server'), ('75e5a760-d61a-4fde-8615-f8c7787d8942', 'allow-incoming-ipv4'), ('e7bbfe53-be00-4120-af51-2670d6b27d69', 'allow-incoming-ipv6'), ('fc203183-4670-425a-a044-4d5438b0e2ac', 'allow-ipv4'), ('674c627f-10f9-44d4-8357-c0a2b81de3d7', 'allow-ipv6'), ('10717f8c-d719-4644-97bc-f3b8825ea5d2', 'clean-traffic'), ('8cbb8b1b-645e-44c0-b171-aecabd8d96c3', 'clean-traffic-gateway'), ('033bdd44-1f85-4abc-9cfd-10049c0ddb8c', 'no-arp-ip-spoofing'), ('bcfcbb1e-693b-4a05-837d-61adebb72805', 'no-arp-mac-spoofing'), ('3a10202a-7685-4a49-9aee-ab0b704fe63e', 'no-arp-spoofing'), ('e42d0d2f-e768-49b1-8ce2-bd5fab8b9bdf', 'no-ip-multicast'), ('192c79dd-0822-4792-9fb8-c2f132b3bc75', 'no-ip-spoofing'), ('f6167dc8-d2a2-491a-b139-544cb454ceb6', 'no-ipv6-multicast'), ('d2d8b32a-91b4-47f0-a15c-f2956ceb2560', 'no-ipv6-spoofing'), ('b06561e1-6132-4748-b8df-59b17ead42a6', 'no-mac-broadcast'), ('e8dedd25-53dc-41f9-8e3f-1deb3b96106a', 'no-mac-spoofing'), ('8332c828-3f9e-4deb-b358-25e52c623271', 'no-other-l2-traffic'), ('5b08c404-dcfe-45b4-b1a8-f92c84f5352a', 'no-other-rarp-traffic'), ('3b1eab43-8976-4b48-99f3-05a645b8e8af', 'qemu-announce-self'), ('b5aa9207-d9d7-4424-a088-7e2d51737743', 'qemu-announce-self-rarp')]
>>> index = uuid.index(('UUID', 'Name'))
>>> remain_uuid = uuid[index+1:]
>>> remain_uuid
[('87c615b3-0f60-4c62-a092-abbce72ca40e', 'allow-arp'), ('8f689e55-b7b3-4d12-a412-e6b09c36eb42', 'allow-dhcp'), ('2c9b69fa-8ffe-4680-be65-0124764a8823', 'allow-dhcp-server'), ('f047154c-a246-47ee-943a-c1ddd8af938a', 'allow-dhcpv6'), ('06f19619-aaf6-4df0-8376-e15191183811', 'allow-dhcpv6-server'), ('75e5a760-d61a-4fde-8615-f8c7787d8942', 'allow-incoming-ipv4'), ('e7bbfe53-be00-4120-af51-2670d6b27d69', 'allow-incoming-ipv6'), ('fc203183-4670-425a-a044-4d5438b0e2ac', 'allow-ipv4'), ('674c627f-10f9-44d4-8357-c0a2b81de3d7', 'allow-ipv6'), ('10717f8c-d719-4644-97bc-f3b8825ea5d2', 'clean-traffic'), ('8cbb8b1b-645e-44c0-b171-aecabd8d96c3', 'clean-traffic-gateway'), ('033bdd44-1f85-4abc-9cfd-10049c0ddb8c', 'no-arp-ip-spoofing'), ('bcfcbb1e-693b-4a05-837d-61adebb72805', 'no-arp-mac-spoofing'), ('3a10202a-7685-4a49-9aee-ab0b704fe63e', 'no-arp-spoofing'), ('e42d0d2f-e768-49b1-8ce2-bd5fab8b9bdf', 'no-ip-multicast'), ('192c79dd-0822-4792-9fb8-c2f132b3bc75', 'no-ip-spoofing'), ('f6167dc8-d2a2-491a-b139-544cb454ceb6', 'no-ipv6-multicast'), ('d2d8b32a-91b4-47f0-a15c-f2956ceb2560', 'no-ipv6-spoofing'), ('b06561e1-6132-4748-b8df-59b17ead42a6', 'no-mac-broadcast'), ('e8dedd25-53dc-41f9-8e3f-1deb3b96106a', 'no-mac-spoofing'), ('8332c828-3f9e-4deb-b358-25e52c623271', 'no-other-l2-traffic'), ('5b08c404-dcfe-45b4-b1a8-f92c84f5352a', 'no-other-rarp-traffic'), ('3b1eab43-8976-4b48-99f3-05a645b8e8af', 'qemu-announce-self'), ('b5aa9207-d9d7-4424-a088-7e2d51737743', 'qemu-announce-self-rarp')]
>>>

**Review by yalzhang — APPROVED** [review]

File: virttest/utils_libvirt/libvirt_nwfilter.py [hunk 1]
```diff
@@ -28,3 +29,21 @@ def clean_up_nwfilter_binding(ignore_status=False):
...
+def get_nwfilter_list():
+    """
+    Get nwfilter list in tuple(UUID, Name)
+
+    Usage:
+         # virsh nwfilter-list
+         UUID                                  Name
+         --------------------------------------------------
+         5c79e80b-cfb5-46d0-9490-77db3318a4b5  allow-arp
+         d8150f0b-2859-4049-a58d-963f33c59aa4  allow-dhcp
```
Comment (chloerh) [inline virttest/utils_libvirt/libvirt_nwfilter.py:43]: This is the usage of virsh nwfilter-list, not the usage of this function. Would you please update with the actual usage, or remove this part?

Comment (chunfuwen) [reply virttest/utils_libvirt/libvirt_nwfilter.py]: @chloerh 
Add more explanation for usage

**Review by chloerh — COMMENTED** [review]
Please ignore my comment. I don't think it's necessary to block this pr, please request another reviewer. Thanks.

File: virttest/utils_libvirt/libvirt_nwfilter.py (hunk 1 above)
Comment (chloerh) [reply virttest/utils_libvirt/libvirt_nwfilter.py]: Please ignore my comment. Thanks.

**Comment by chunfuwen** [comment]
@dzhengfy 
please help take a look

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**